### PR TITLE
Release version update

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5921

*Description of changes:*

* Release version update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2665,6 +2665,66 @@
         "tslint": "^5.20.1",
         "webpack": "^4.0.0",
         "yorkie": "^2.0.0"
+      },
+      "dependencies": {
```
Comment (severinbeauvais) [inline package-lock.json:2669]: I think this is what happens if you don't use the correct node/npm version. Try again with:
```
$ node --version
v14.15.0
$ npm --version
6.14.9
```

**Review by severinbeauvais — CHANGES_REQUESTED** [review]
I'm concerned about the package lock diff.

File: package-lock.json (hunk 1 above)
Comment (cameron-freshworks) [reply package-lock.json:2669]: kk, will do

Comment (cameron-freshworks) [reply package-lock.json]: Updated!

**Comment by severinbeauvais** [comment]
obsolete change

**Pull request closed without merging**